views/pages/allJobRolesList.html

```diff
@@ -21,6 +22,7 @@ <h1>Job Roles</h1>
             <td>{{ jobRole.capability }}</td>
             <td>{{ jobRole.band }}</td>
             <td>{{ jobRole.closingDate | date }}</td>
+            <td><a href="/job-apply-{{jobRole.id}}" class="btnKainos">Apply Now</a></td>
```

Comment:
only an applicant user can apply for roles, not recruitment admin so this button should only show depending on role